nix/desktop/macos/default.nix

```diff
@@ -11,6 +11,7 @@ let
 
 in {
   buildInputs = [
+    pkgs.gnupg22
```

Comment:
@jakubgs why aren't we receiving `gnupg22` instead of `pkgs` in the argument list. We should avoid passing `pkgs` as much as possible, and in this case it doesn't seem to be required.

Reply:
Good point, I'll fix it in another PR.